options/glibc/include/sys/ioctl.h

```diff
@@ -25,6 +25,9 @@ int ioctl(int fd, unsigned long request, ...);
 #define SIOCSIFFLAGS 0x8914
 #define SIOCGIFINDEX 0x8933
 
+#define SIOCDEVPRIVATE 0x89F0
+#define SIOCPROTOPRIVATE 0x89E0
```

Comment:
We sort these by value from what I can tell. Swap this one around with the other one to match that.